assets/src/scss/layout/_page-header.scss

```diff
@@ -79,27 +79,40 @@
 }
 
 .page-header-background {
+  overflow: hidden;
   position: absolute;
   z-index: 0;
   top: 0;
   right: 0;
   left: 0;
 
+  _-- {
+    bottom: auto;
+  }
+
   &:after {
     content: "";
     display: block;
     position: absolute;
     z-index: 0;
-    height: 75%;
-    bottom: -.5%;
     left: 0;
     right: 0;
-    @include linear-gradient(360deg, $white 0%, transparent 100%);
+
+    _-- {
+      height: 75%;
+      bottom: -.5%;
+      background: linear-gradient(360deg, $white 0%, transparent 100%);
```

Comment:
In other places we don't use the mixin that was previously used. I checked browser support and the fallbacks are not needed anymore. https://caniuse.com/?search=linear-gradient